File: GTBIT/L001_Foundation/l001_Basic/l001.java
import java.util.Scanner;
public class l001{
    public static Scanner scn = new Scanner(System.in);
    public static void printHello(){
        System.out.println("Hi, There");
    }
    public static void takeInput(){
        int n = scn.nextInt();
        System.out.println("Your number "+ n);
    }
    public static void workWithDataType(){
        int a=10;
        int b=20;
        System.out.println("Value of A: "+ a);
        System.out.println("Value of B: "+ b);
    }
    public static void main(String[] args){
        // printHello();
        // workWithDataType();
        takeInput();
    }
}

File: GTBIT/L001_Foundation/l001_Basic/l001.cpp
#include<bits/stdc++.h>
using namespace std;

void printHello(){
    cout<<"Hi There"<<endl;
}
void workWithDataType(){
    int a=10;
    int b=20;
    cout<<"C++ +\n";
    cout<<"Value of A: " + to_string(a) <<endl;
    cout<<"Value of B: " + to_string(b) <<endl;
}
void takeInput(){
    int n; cin>>n;
    cout<<"your number "+to_string(n)<<"\n";
}
int main(){
    // printHello();
    // workWithDataType();
    takeInput();
    return 0;
}